### DISCUSSION TITLE
如何导出日志？

### DISCUSSION THREAD

**Discussion by sqq135** [hooke007/mpv_PlayKit#433] [Q&A 问答]
平时播放非常顺滑，突然播放个番剧换集时会闪退，其他番剧都不会，如何输出日志查看问题啊？（闪退时控制台也没有反应）

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/blob/2a681161e227e2b56af3b3882149713e0513b6ef/portable_config/mpv.conf#L10

**Reply by sqq135** [reply]
[
[mpv-lazy.log](https://github.com/hooke007/MPV_lazy/files/14703703/mpv-lazy.log)
](url)

感谢指导，之前从没遇到过这样，切换剧集的时候闪退。怀疑是视频的问题，不知道log能不能看出来。

其他的视频都正常，好像这是个例。

**Reply by hooke007** [reply]
日志中断。复数变量太多了，不好推测。
把一些实验性的和动态的相关的去掉

**Reply by sqq135** [reply]
[mpv-lazy.log](https://github.com/hooke007/MPV_lazy/files/14717513/mpv-lazy.log)

感谢指导，经过排查，发现是profile的问题，试过之后发现是这个，可能是之前的设置又保留了下来。顺便问下，profile能不能直接抄作业，code代码那个。

[native_hwdec_auto]
 profile-desc = 超过2k宽度的片源自动dx纯硬解
 profile-cond = width>=2000
 profile-restore = copy
 hwdec = vulkan-copy

**Reply by hooke007** [reply]
你这个描述和参数就不是一个东西啊，vulkan-copy切的时候崩了吧，这东西一直在实验性测试。

抄哪个？

**Reply by sqq135** [reply]
哈，一开始用的是个net的分支，转移到lazy时保留了一部分，所以出问题了。抄作业我是说这个https://github.com/hooke007/MPV_lazy/blob/main/portable_config/profiles.conf

“你这个描述和参数就不是一个东西啊”这个啥意思，反正我删掉这个就解决了。

准备再把你写的profiles添上去，感谢指导

**Reply by hooke007** [reply]
vulkan-copy 是vulkan的复制解码，和“dx纯硬解”没关系。

主仓库的配置文件主要是当说明书和示例用，具体实际怎么应用看场景调节。
